docs/docs/docs/01-core/admin/01-ee/02-audit-logs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Audit Logs
+description: Guide for Audit Logs
+tags:
+  - settings
+  - audit-logs
+---
+
+# Audit Logs
+
+## Summary
+
+Audit Logs provide a way to view the history of all user actions at Admin API level. This includes actions related to entries (including publish actions), media (including its folders), users, login & logout of admin users, components, roles, and permissions, you can see the list of all the default events [here](https://github.com/strapi/strapi/blob/main/packages/core/admin/ee/server/services/audit-logs.js#L9).
+
+## Backend design
+
+The Audit Logs feature was built to take advantage of the eventHub. You can find the service and its core code at `packages/core/admin/ee/server/services`.
```

Comment:
How is it stored in db? Could you also mention the audit log providers?

It's a good pattern we are using and it could help understand how everything is connected together